sensor_msgs/PointCloud2 cloud
sensor_msgs/PointCloud2 rgb_cloud
sensor_msgs/Image rgb_img
sensor_msgs/CameraInfo rgb_info
bool stop_publish
---
bool result
